code/object/objectsort.cpp

```diff
@@ -235,7 +235,8 @@ void obj_render_all(const std::function<void(object*)>& render_function, bool *d
 		if ( (obj == Viewer_obj)
 			&& (obj->type == OBJ_SHIP)
 			&& c_viewer
-			&& (Ship_info[Ships[obj->instance].ship_info_index].flags[Ship::Info_Flags::Show_ship_model]) )
+			&& (Ship_info[Ships[obj->instance].ship_info_index].flags[Ship::Info_Flags::Show_ship_model])
+			&& (!Show_ship_only_if_cockpits_enabled || (Show_ship_only_if_cockpits_enabled && Cockpit_active)) )
```

Comment:
This line can just be simplified to
``&& (!Show_ship_only_if_cockpits_enabled || Cockpit_active)``

Reply:
Thanks, just updated with that simplification